cmd/dockerregistry/main.go

```diff
@@ -16,6 +18,18 @@ import (
 )
 
 func main() {
+	// Start the profiling server, if requested.
+	profilingAddr := os.Getenv("REGISTRY_PROFILING_ADDR")
```

Comment:
Use the same env we use on Openshift binaries, both address and the web
bit.  In fact, just use the serviceability package.

On Mar 4, 2016, at 9:34 AM, Dan Mace <notifications@github.com> wrote:

In cmd/dockerregistry/main.go
<https://github.com/openshift/origin/pull/7771#discussion_r55036218>:

> @@ -16,6 +18,18 @@ import (
>  )
>
>  func main() {
> +	// Start the profiling server, if requested.
> +	profilingAddr := os.Getenv("REGISTRY_PROFILING_ADDR")

That's interesting. I tested this, and the registry definitely didn't
terminate. I'll change it regardless.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/7771/files#r55036218>.